### ISSUE TITLE
test failures: missing strict_permissions attr

### ISSUE THREAD

**Issue by diracdeltas** [letsencrypt/letsencrypt#803]
When I run `tox -e py27` inside Docker, I get these errors:

```
======================================================================
ERROR: test_it (letsencrypt.tests.crypto_util_test.InitSaveCSRTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/letsencrypt/src/.tox/py27/local/lib/python2.7/site-packages/mock.py", line 1201, in patched
    return func(*args, **keywargs)
  File "/opt/letsencrypt/src/letsencrypt/tests/crypto_util_test.py", line 65, in test_it
    'csr-letsencrypt.pem')
  File "/opt/letsencrypt/src/letsencrypt/crypto_util.py", line 83, in init_save_csr
    config.strict_permissions)
  File "/opt/letsencrypt/src/letsencrypt/configuration.py", line 39, in __getattr__
    return getattr(self.namespace, name)
AttributeError: '_AttrDict' object has no attribute 'strict_permissions'

======================================================================
ERROR: test_success (letsencrypt.tests.crypto_util_test.InitSaveKeyTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/letsencrypt/src/.tox/py27/local/lib/python2.7/site-packages/mock.py", line 1201, in patched
    return func(*args, **keywargs)
  File "/opt/letsencrypt/src/letsencrypt/tests/crypto_util_test.py", line 37, in test_success
    key = self._call(1024, self.key_dir)
  File "/opt/letsencrypt/src/letsencrypt/tests/crypto_util_test.py", line 32, in _call
    return init_save_key(key_size, key_dir, 'key-letsencrypt.pem')
  File "/opt/letsencrypt/src/letsencrypt/crypto_util.py", line 53, in init_save_key
    config.strict_permissions)
  File "/opt/letsencrypt/src/letsencrypt/configuration.py", line 39, in __getattr__
    return getattr(self.namespace, name)
AttributeError: '_AttrDict' object has no attribute 'strict_permissions'

----------------------------------------------------------------------
Ran 391 tests in 7.805s

FAILED (errors=2)
ERROR: InvocationError: '/opt/letsencrypt/src/.tox/py27/bin/python setup.py test -q'
```

**Comment by diracdeltas** [comment]
git blame suggests cc'ing @pde

**Comment by jdkasten** [comment]
Fixed in #777.  I ran into this problem while working on #777.  I fixed it there... though I know I should have probably pulled it out into a separate PR.  Luckily, it is ready for review.

**Comment by jdkasten** [comment]
Fixed it in #777
